packages/postgrest/lib/src/postgrest_builder.dart

```diff
@@ -398,7 +415,25 @@ class PostgrestBuilder<T, S> implements Future<T> {
 
     try {
       final response = await _execute();
-      final data = response.data;
+      var data = response.data;
+
+      // Workaround for https://github.com/supabase/supabase-flutter/issues/560
```

Comment:
Isn't this already done in `_parseResponse()`?

Reply:
Oops, I meant to take this one out. Thanks for catching it!